journal/journal/views.py

```diff
@@ -0,0 +1,85 @@
+import transaction
+from pyramid.response import Response
+from pyramid.view import view_config
+from pyramid.httpexceptions import HTTPFound
+from wtforms import Form, StringField, TextAreaField, validators
+from jinja2 import Markup
+import markdown
+
+from sqlalchemy.exc import DBAPIError
+
+from .models import (
+    DBSession,
+    Entry,
+    )
+
+
+class NewEntry(Form):
+    title = StringField('title', [validators.Length(min=1, max=128)])
+    text = TextAreaField('text')
+
+
+@view_config(route_name='home', renderer='templates/home.jinja2')
+def my_view(request):
+    try:
+        all_entries = DBSession.query(Entry).order_by(Entry.id.desc()).all()
+        return {'entries': all_entries}
+    except DBAPIError:
+        return Response("home broke", content_type='text/plain', status_int=500)
```

Comment:
You can omit lines 27/28 here.  They don't do anything for you that wouldn't happen anyway when the DBAPIError raises without being handled.  Moreover, it might cause problems if the session you are working in is in a bad order.  The raised exception would abort and roll it back.  Swallowing it like this would not do so.